cim-forward-route/src/main/resources/application.yaml

```
@@ -70,3 +70,26 @@ offline:
+  endpoints:
+    web:
+      exposure:
+        include: health,info,metrics
+
+# OpenTelemetry gRPC endpoint for SDK exporter
+otel:
+  exporter:
+    otlp:
+      grpc:
+        endpoint: ${OTEL_EXPORTER_OTLP_GRPC_ENDPOINT:http://localhost:4317}
+
```

YAMLlint (1.38.0)

[warning] too many blank lines (1 > 0)

(empty-lines)